src/plugins/intel_cpu/src/graph_dumper.cpp

```diff
@@ -39,7 +43,19 @@ std::map<std::string, std::string> extract_node_metadata(const NodePtr &node) {
         // Path to print actual name for extension layers
         serialization_info[ExecGraphInfoSerialization::LAYER_TYPE] = node->getTypeStr();
     } else {
-        serialization_info[ExecGraphInfoSerialization::LAYER_TYPE] = NameFromType(node->getType());
+        std::string layerTypeStr;
+
+        auto layerType = node->getType();
+
+        /* replace CPU proprietary input/output types with the ones which serializer can process */
+        if (layerType == Type::Input)
+            layerTypeStr = "Parameter";
+        else if (layerType == Type::Output)
+            layerTypeStr = "Result";
```

Comment:
This is necessary, because ngraph serializer looks for those node types in order to serialize TensorIterator inputs and outputs correctly. Basically there is two options:
1) Current one
2) Update ngraph serializer with "Input" and "Output" types support

Reply:
See https://github.com/openvinotoolkit/openvino/blob/master/src/core/src/pass/serialize.cpp#L391